src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -73,7 +73,12 @@
     </li>
     <li>
       <p>
-        In the prompt that appears, select <strong>Allow</strong> to open a settings page.
+        In the prompt that appears, select <strong>Allow</strong> and then <strong>Close</strong>.
```

Comment:
What do you think of adding "to download the profile" after "Allow" and adding "select" before "Close"? No big deal either way; I just thought that might add some context about what's happening between the selections.